integration-cli/docker_cli_cp_test.go

```diff
@@ -472,7 +472,7 @@ func TestCpVolumePath(t *testing.T) {
 	}
 
 	cleanedContainerID := strings.TrimSpace(out)
-	defer deleteContainer(cleanedContainerID)
+	defer dockerCmd(t, "rm", "-fv", cleanedContainerID)
```

Comment:
Yes, but what it is causing the 2nd one to work isn't the TERM vs KILL, it's that `-f` has tells Docker to actually remove a container that is in a `dead` state no matter what.